05/style.css

```diff
@@ -0,0 +1,110 @@
+* {
+	padding: 0;
+	margin: 0;
+	box-sizing: border-box;
+}
+
+/* base style */
+.container {
+	margin: 0 auto;
+	max-width: 1000px;
+	width: 100%;
+	border: 1px solid green;
+	margin: 2px;
+}
+
+.item {
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	margin: 5px;
+	border: 1px solid rgb(43, 43, 43);
+	background-color: blue;
+}
+
+/* mobile */
+.nav {
+	display: grid;
+	grid-template-columns: repeat(1, 1fr);
```

Comment:
Osobiście użyłbym flex-a - będzie prościej. `grid` wtedy kiedy mamy kilka wierszy i kolumn, ale może zostać to nie błąd ;)